### ISSUE TITLE
Allow using this library with Uint8Array

### ISSUE THREAD

**Issue by webmaster128** [crypto-browserify/pbkdf2#79]
Would it be possible to except not only Buffer and string for password/salt but also Uint8Array?

All my crypto code uses Uint8Array for both browser and nodejs and it would be cool to be able to use the JS standard type directly.

**Comment by calvinmetcalf** [comment]
yeah looks like default node does support that so we should probably include that

**Comment by abhijithvijayan** [comment]
Will this be resolved?
